### DISCUSSION TITLE
Android minSDK

### DISCUSSION THREAD

**Discussion by thebino** [AndroidDev-social/DodoForMastodon#20] [General]
The MVP has currently a **minSDK 23**
On Slack and in PR Reviews some thoughts where raised.
So let's move the discussion into here instead.

**Comment by thebino** [comment]
Support as low as possible but try to reduce the maintenance only to cover legacy versions.
Always keep in mind, there are regions on earth, where not the latest phone is in everyone’s pocket.

**Comment by thebino** [comment]
minSDK 26 has a target audience of 88.2% right now

**Comment by thebino** [comment]
Java 8 datetime needs at least minSDK 26 (w/out desugaring)

https://tunesoftware.com/apps/java-8-date-time-classes-in-android-below-api-26/

**Reply by racka98** [reply]
Last time I was setting up desugaring I had to add it in every module that used kotlinx-datetime and there were some instances were the app still crashed regardless. I just gave up and moved to sdk 26. Since desugaring also has an impact on build times.

**Comment by roshanrai06** [comment]
If you see the app reach and target audience  26 is fine 88.2% is good, plus our target audience will be mostly younger. It will allow us to use the latest frameworks and fewer legacy OS bugs, I have seen some bugs in the older device which are very hard to fix, plus very hard to find that device for the test as well!!

The current distribution of 88.2% for API 26, and by the time our app is released this will be increased significantly. However we will take a collective decision on this, so let's for 
others' opinions as well :)

**Comment by racka98** [comment]
Though sdk 23 will still be fine. Just will have a couple of checks for most of stuff updated in 8.0 like new Notifications api, vibration manager, camera and storage stuff, etc
